prog/rand.go

```diff
@@ -13,7 +13,8 @@ import (
 	"strings"
 
 	"github.com/google/syzkaller/pkg/ifuzz"
-	_ "github.com/google/syzkaller/pkg/ifuzz/generated" // pull in generated instruction descriptions
+	"github.com/google/syzkaller/pkg/ifuzz/ifuzzimpl"
+	_ "github.com/google/syzkaller/pkg/ifuzz/x86/generated" // pull in generated instruction descriptions
```

Comment:
My idea was that nobody outside of pkg/ifuzz imports x86/generated nor ifuzzimpl (thus "impl" in the name).
Let me see if I can shuffle the code that way.